src/tests/__init__.py

```diff
@@ -0,0 +1 @@
+"""Test for pydocstyle."""
```

Comment:
py.test does not require an `__init__.py` and often recommends you don't make your tests module importable.